storage/src/main/java/tech/pegasys/artemis/storage/MapDbDatabase.java

```diff
@@ -257,10 +256,10 @@ private void pruneCheckpointStates(final Checkpoint newFinalizedCheckpoint) {
 
   private void pruneHotBlocks(final Checkpoint newFinalizedCheckpoint) {
     // TODO: Can we prune blocks from in the finalized epoch as well?
```

Comment:
I suspect we don't need this TODO anymore - we can prune anything up to the finalised block.